spec/mailers/enrollment_mailer_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe EnrollmentMailer, type: :mailer do
+  before do
+    ActsAsTenant.current_tenant = Instance.default
+  end
+
+  let!(:course) { create(:course) }
+  let!(:student) { create(:user) }
+  let!(:lecturer) { create(:course_owner, course: course, user: course.creator, status: :approved) }
+  let!(:registration) { create(:course_student, course: course, user: student) }
+
+  def translate(mail_action, mail_field)
+    I18n.t("enrollment_mailer.#{mail_action}.#{mail_field}", raise: true)
```

Comment:
This doesn't seem right either... I don't think it's meant to be used like this :P